README.md

```diff
@@ -76,7 +76,21 @@ Also for some notes specific to this collection see [our CONTRIBUTING documentat
 
 See [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#testing-collections).
 
-### Communication
+## Collection maintenance
+
+To learn how to maintain / become a maintainer of this collection, refer to:
+
+* [Committer guidelines](https://github.com/ansible-collections/community.general/blob/main/commit-rights.md).
+* [Maintainer guidelines](https://github.com/ansible-collections/community.mysql/blob/main/MAINTAINING.md).
```

Comment:
Do you mean https://github.com/ansible/community-docs/blob/main/maintaining.rst, or the file in community.mysql?

Reply:
I wish community.mysql would be a keystone collection but in this case i meant the maintaining.rst file:) thanks for spotting! fixed. Ready for review:)